web-service-consumer/1.5/modules/ROOT/pages/index.adoc

```diff
@@ -1,5 +1,4 @@
 = Web Service Consumer Connector- Mule 4
-:page-aliases: connectors::web-service/web-service-consumer.adoc
```

Comment:
Here are my comments for this section:
Important: Remove the POM file information since it is now in the XML topic

Not important for now: Remove periods in the list of features.
We are adding a link to the introduction to connectors topic the See Also section of every topic now. We are also adding a link to the introductory Exchange topic in the index.adoc topic. I thik these are in  the template.